src/components/nimbleButton/NimbleButton.tsx

```diff
@@ -10,10 +10,12 @@ interface ButtonProps {
   buttoncolor: string;
   fontFamily: string;
   fontWeight: string;
+  labelColor: string;
```

Comment:
Please add this as an optional prop. If not after updating the lib all the projects have to be updated.
If prop is not there don't add the label color to styled component put undefined